README.md

```diff
@@ -82,11 +82,13 @@ Diagrams with an overview of the OpenSSF, including its projects and SIGs, are a
 | Package Analysis       | https://github.com/ossf/package-analysis         | [Meeting Notes](https://docs.google.com/document/d/1GFslP6elYCx27TUitdigDr1gsOItYkL0Vq7hTB9y4Lo/edit) | Securing Critical Projects WG   | TBD        |
 | Package Feeds          | https://github.com/ossf/package-feeds            | [Meeting Notes](https://docs.google.com/document/d/1GFslP6elYCx27TUitdigDr1gsOItYkL0Vq7hTB9y4Lo/edit) | Securing Critical Projects WG   | TBD        |
 | Repository Service for TUF | https://github.com/repository-service-tuf/repository-service-tuf |  [Meeting Notes](https://docs.google.com/document/d/13a_AtFpPK9WO4PlAN6ciD-G1jiBU3gEDtRD1OUinUFY/edit)  | Securing Software Repositories WG | Sandbox |
+| S2C2F                  | https://github.com/ossf/s2c2f                    | [Meeting Notes](https://docs.google.com/document/d/10Q_VOvKsGaYJoK-5yJY4868mTkYZjEo-6xV6ghYS84k/edit#heading=h.1tv8gumsrtbf) | Supply Chain Integrity WG  | TBD      |
```

Comment:
Minor: S2C2F's repo refers to itself as a SIG, but I think based on our work last year expanding https://github.com/ossf/tac/blob/main/process/project-lifecycle.md to cover projects we think it should be classified as a project instead? That's fine with me, just want to clarify!